SITCOMTN-086.tex

```diff
@@ -46,24 +62,823 @@
+of measuring the relative response of all the billions of pixels in LSSTCam on
+small scales.
+
+The dome flat is a good start for estimating the reference flax flat, but has
+issues at small scales and large scales.  At small scales, there are lateral
+electric field effects such as ``tree rings'' and the ``picture frame'' that
+are pixel size variations that show up as apparent QE variations in the dome
+flat.  At large scales, the response to focused light is different than the
+dome flat which includes unfocused and scattered light.
+
+To handle lateral electric field effects we fit templates based on our physical
+knowledge of the detectors.  This includes radial templates for tree rings with
+a fixed center, and square templates around the edge of the detectors.  The
+exact methodology is TBD.
+
+To handle large scale variations for LSST Y1 we use a ``star flat'' generated
+from a sequence of multi-band dithered observations of a moderately dense star
+field.  These observations are cross-calibrated using FGCM to generate
+per-detector polynomial approximations of the throughput relative to the dome
+flat at scales of hundreds of pixels and greater.  By making use of stars of
+various colors, the FGCM solution can approximate the star flat terms for a
+flat reference spectrum.
+
+In the future, we plan to use what we call the ``dynamic'' CBP, where the
+synthetic stars from the CBP are dithered around the full focal plane.  The
+advantage of this is that the response can be more carefully controlled, and it
+can be performed in narrow wavelength steps to build up the dynamic CBP star
+flat with any arbitrary spectrum.  The software to perform this analysis will
+not be ready by the completion of construction.
+
+\subsection{Atmosphere Response}
+
+The attenuation of light due as it propagates through the atmosphere includes
+absorption and Rayleigh scattering from molecular constituents (primarily $O_2$
+and $O_3$); absorption by preciptable water vapor (PWV), Mie scattering by
+airborne particulate aerosols that have physical sizes comparable to the
+wavelength of visible light; and shadowing by ice crystals and water droplets
+in clouds that is independent of wavelength.
+
+This can be summarized with the following equation:
+%
+\begin{equation}
+S^{\mathrm{atm}}(\mathrm{alt}, \mathrm{az}, t, \lambda) =
+S^{\mathrm{molecular}}(\mathrm{bp}, \mathrm{zd}, t,
+\lambda)\,S^{\mathrm{PWV}}(\mathrm{zd}, t,
+\lambda)\,e^{-(X(\mathrm{zd})\,\tau(t, \lambda))},
+\end{equation}
+%
+where $S^{\mathrm{molecular}}$ is the absorption and scattering from dry gasses
+(primarily in the u, g, and r bands); $S^{\mathrm{PWV}}$ is the absorption by
+water vapor (primarily in the z and y bands); $\tau$ is the aerosol optical
+depth (AOD); $X$ is the airmass; and $\mathrm{bp}$ is the barometric pressure.
+
+Following FGCM, we obtain the molecular scattering using MODTRAN (ref) combined
```

Comment:
From http://modtran.spectral.com/modtran_faq#what_ref

What reference(s) should be used for MODTRAN?
Recent MODTRAN publications include [Berk et al., 2014; 2015]. The 2014 paper should be used as a general MODTRAN reference, while the 2015 paper focuses on the MODTRAN line-by-line algorithm:

1. A. Berk, P. Conforti, R. Kennett, T. Perkins, F. Hawes, and J. van den Bosch, ["MODTRAN6: a major upgrade of the MODTRAN radiative transfer code,"](http://proceedings.spiedigitallibrary.org/proceeding.aspx?articleid=1882598) Proc. SPIE 9088, Algorithms and Technologies for Multispectral, Hyperspectral, and Ultraspectral Imagery XX, 90880H (June 13, 2014); doi:10.1117/12.2050433.

2. Alexander Berk, Patrick Conforti, and Fred Hawes, ["An accelerated line-by-line option for MODTRAN combining on-the-fly generation of line center absorption with 0.1 cm-1 bins and pre-computed line tails,"](http://spie.org/Publications/Proceedings/Paper/10.1117/12.2177444) Proc. SPIE 9471, Algorithms and Technologies for Multispectral, Hyperspectral, and Ultraspectral Imagery XXI, 947217 (May 21, 2015); doi:10.1117/12.2177444